src/hooks/marp-options.ts

```diff
@@ -9,8 +9,9 @@ interface MarpReactOptions {
 }
 
 const chars = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz'
+const nanoid = customAlphabet(chars, 8)
```

Comment:
This is due to the change in `nanoid v3` (c.f. https://github.com/ai/nanoid/releases/tag/3.0.0)